exampleCourse/questions/element/markdown/question.html

```diff
@@ -1,6 +1,3 @@
-<!--htmlhint tag-pair:false -->
-<!-- htmlhint doesn't understand our <markdown> tag-->
```

Comment:
Pointing out that the regex-based search for `<markdown>` actually used this tag (not the one a couple of lines below) as the starting point for markdown processing. unified ignored the `-->`, but marked is escaping it, causing a problem with the termination of this comment. There is a small chance that this could break existing questions... Should this be handled separately?